### DISCUSSION TITLE
Project delete & S3 buckets [Jun 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5992] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### - Deleting records from grid view will clear records   
#### - Deleting projects & deleting attachments as you have noticed is currently soft-delete.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1110521704247201833)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
